seqr/views/apis/report_api.py

```diff
@@ -398,12 +401,12 @@ def _add_row(row, family_id, row_type):
         get_additional_individual_fields=_get_participant_row,
         post_process_variant=_post_process_gregor_variant,
         include_svs=False,
-        airtable_fields=[SMID_FIELD, PARTICIPANT_ID_FIELD, 'Recontactable'],
+        airtable_fields=[[PARTICIPANT_ID_FIELD, 'Recontactable'], [SMID_FIELD]],
```

Comment:
it might be cleaner to use split this into two variables, `airtable_fields` and `airtable_list_fields` or something like that rather a list with two elements.

Reply:
I thought of that, but this gets passed through about 5 different places so it made or a kind of messy signature change to a lot of unrelated functions and I thought that messiness was not worth the trade off

Reply:
🤷 yeah, could do a `namedtuple` or something along those lines but doesn't matter much!